packages/create-pantheon-decoupled-kit/src/actions/runLint.ts

```diff
@@ -27,6 +27,13 @@ export const runLint: Action = async ({ data }) => {
 			assert: { type: 'json' },
 		});
 		// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+		if (pkg?.scripts['build-scripts']) {
+			execSync(`${command}  build-scripts`, {
+				cwd: data.outDir,
+				stdio: 'inherit',
+			});
+		}
```

Comment:
Can you move the `npm run build-scripts` to the `next lint` in the package.json instead? If possible, we should keep the actions generic and move the starter specific logic into the package.json scripts.

Reply:
Ah yes definitely, that is a more clever way to do it.